FacuReartes/react-laba-final-project-team-2
# SPT2-189 - Order history page UI functionality

# Description


## Changes

- Updated checkout payment card method.
- Added new after payment method to checkout.
- Created Order History Page.

### Jira

- [SPT2-189](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-189?atlOrigin=eyJpIjoiOTdlNGM4ZjEzZDI5NDMwNWIyOTUxMmYzYjkxMWE4ZjUiLCJwIjoiaiJ9)

### How to Test

1. Navigate to `Order History` and check list of orders made.
2. Go to home page and select new products to bag.
3. Then checkout to see payment methods.
4. After complete the order, go to order history again to see updated list

### Screenshots (if applicable)

![image](https://github.com/user-attachments/assets/34fdd605-2aa5-42d9-bc59-0caa5b668f15)
![image](https://github.com/user-attachments/assets/56e2e8ca-354a-48d2-ab9b-8374a8f45712)


### Checklist



[SPT2-189]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-189?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
